canary/config.go

```diff
@@ -49,6 +49,9 @@ const (
 	CadenceServiceName = "cadence-frontend"
 	// CanaryServiceName is the default service name for cadence canary
 	CanaryServiceName = "cadence-canary"
+	// CrossClusterCanaryModeFull is a canary testing mode which tests all permutations of
+	// the cross-cluster/domain feature
+	CrossClusterCanaryModeFull = "test-all"
```

Comment:
Maybe list all possible options in a comment in the case developers want to change this value in the future?

Reply:
This is the only option so far, but I just didn't want to make it a bool in case there's future options